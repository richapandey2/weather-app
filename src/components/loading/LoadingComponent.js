import React from "react";
import { CircularProgress, Backdrop, makeStyles } from "@material-ui/core";

function LoadingComponent() {
  const useStyle = makeStyles(() => ({
    backdrop: {
      backgroundColor: "#8e898938",
      color: "#fff",
    },
  }));
  const classes = useStyle();
  return (
    <Backdrop className={classes.backdrop} open>
      <CircularProgress color="primary" />
    </Backdrop>
  );
}

export default LoadingComponent;
