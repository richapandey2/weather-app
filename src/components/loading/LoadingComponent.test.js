import React from "react";
import { render, cleanup } from "@testing-library/react";
import LoadingComponent from "./LoadingComponent";

afterEach(cleanup);

test("should test Loading component", () => {
  const { asFragment } = render(<LoadingComponent />);
  expect(asFragment).toMatchSnapshot();
});
