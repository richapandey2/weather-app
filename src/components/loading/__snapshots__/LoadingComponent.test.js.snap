// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Loading component 1`] = `[Function]`;
